### PR TITLE
Converted to using plummet for tests

### PR DESCRIPTION
**Pull request by dusktreader**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, 3.10]
```
Comment (jvllmr) [inline .github/workflows/main.yml]: GitHub provides Python 3.10 as version 3.10.0

Comment (dusktreader) [reply .github/workflows/main.yml]: What's the difference?

Comment (dusktreader) [reply .github/workflows/main.yml]: Oh...it's reading 3.10 as 3.1...ooof

**Comment by dusktreader** [comment]
Investigating why CI pipeline is failing. It was complaining about not being able to find pytest 6.2.5, which is...weird.

**Pull request merged**